com.github.Bavarder.Bavarder.json

```diff
@@ -0,0 +1,54 @@
+{
+    "app-id" : "com.github.Bavarder.Bavarder",
```

Comment:
should `io.github`

Reply:
Why ? I don't have a website

Reply:
https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
So that will be Io.github.Bavarder.Bavarder ?